src/test/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalControllerTest.java

```diff
@@ -186,14 +191,15 @@ public void test_RetrieveOrganisationByIdentifier() {
     @Test
     public void test_RetrievePaymentAccountByUserEmail() {
         final HttpStatus expectedHttpStatus = HttpStatus.OK;
+        RequestContextHolder.setRequestAttributes(new ServletRequestAttributes(httpRequest));
 
         List<String> authorities = new ArrayList<>();
         authorities.add(TestConstants.PUI_USER_MANAGER);
         String email = "test@email.com";
+        when(httpRequest.getHeader(anyString())).thenReturn(email);
```

Comment:
please add `verify()` for this `when().thenReturn()`

Reply:
done